Dataset.py

```diff
@@ -568,6 +599,10 @@ def get_current_seq_order(self):
     raise OptionalNotImplementedError
 
   def _base_init(self):
+    """
+    TODO: seems to be only used in initialize and one location SprintDataset, might be moved
```

Comment:
I think I put it there, so that other datasets potentially could also use it. I think it covers some more exotic cases, so normally it does not matter whether this is called or not. And it might also be too specific for `SprintDataset` for historical reasons but not sure. I don't remember the details anymore. But when I look at the code, I guess that most other datasets handle `window` wrongly (or not at all), and also `nbytes` or `zpad` are probably not correctly handled (but also that might not really be relevant in most cases).